src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.tsx

```diff
@@ -0,0 +1,229 @@
+import MyDataHelps, { DataCollectionSettings, ExternalAccount } from '@careevolution/mydatahelps-js';
+import React, { ReactElement, useEffect, useState } from 'react';
+import ConnectDevicesIcon from '../../../assets/connectdevices.svg';
+import FitbitIcon from '../../../assets/fitbit.svg';
+import GarminIcon from '../../../assets/garmin.svg';
+import AppleHealthIcon from '../../../assets/appleHealth.svg';
+import GoogleFitIcon from '../../../assets/googleFit.svg';
+import OmronIcon from '../../../assets/omron.png';
+import { Action, Title } from '../../presentational';
+import "./ConnectDevicesMenu.css"
+import { FontAwesomeSvgIcon } from 'react-fontawesome-svg-icon';
+import { faRefresh } from '@fortawesome/free-solid-svg-icons';
+import { getFitbitProviderID, getGarminProviderID, getOmronProviderID } from '../../../helpers/providerIDs';
+import { previewAccounts, previewSettings } from './ConnectDevicesMenu.previewdata';
+import language from '../../../helpers/language';
+
+export type DeviceAccountType = "Fitbit" | "Garmin" | "AppleHealth" | "GoogleFit" | "Omron";
+
+export interface ConnectDevicesMenuProps {
+    innerRef?: React.Ref<HTMLDivElement>
```

Comment:
What is `innerRef` used for?

Reply:
It's so that any consumers of the library can attach a ref to the component; in particular needed for the view builder to work correctly since this component would need to be draggable/droppable without a wrapper div